runtime/dolphin/src/lib.rs

```diff
@@ -582,6 +590,92 @@ impl pallet_preimage::Config for Runtime {
 	type ByteDeposit = PreimageByteDeposit;
 }
 
+parameter_types! {
+	// Does not really matter as this will be only called by root
+	pub const AssetDeposit: Balance = 0;
+	pub const AssetAccountDeposit: Balance = 0;
+	pub const ApprovalDeposit: Balance = 0;
+	pub const MetadataDepositBase: Balance = 0;
+	pub const MetadataDepositPerByte: Balance = 0;
```

Comment:
Can you add a FIXME here? I think these constants need to be revisited here.

Reply:
I've added them to this issue https://github.com/Manta-Network/Manta/issues/386
Btw the comment here is still correct. These don't really matter because only Root can call the extrinsics.